### ISSUE TITLE
Indentation after an decorator in an example

### ISSUE THREAD

**Issue by Ulrikop** [gajus/eslint-plugin-jsdoc#789]
## Expected behavior

Examples that use a decorator allow additional indentation, such as an example without a decorator. 

## Actual behavior

An example that has an additional indentation on a new line after a decorator throws `There must be no indentation`.

## ESLint Config

```js
{
  "env": {
    "es6": true
  },
  "parser": "@typescript-eslint/parser",
  "plugins": ["jsdoc"],
  "rules": {
    "jsdoc/check-indentation": "error"
  }
}
```

## ESLint sample

```js
/**
 * @example 
 * ```
 * @MyDecorator({
 *   myOptions: 42
 * })
 * export class MyClass {}
 * ```
 *
 * @example 
 * ```
 * MyDecorator({
 *   myOptions: 42
 * })
 * ```
 */
function MyDecorator(options: { myOptions: number }) {
  return (Base: Function) => {};
}
```

First `@example` throw `There must be no indentation`, the second `@example` is fine.

## Environment

- Node version: 12.18.4
- ESLint version 7.32.0
- `eslint-plugin-jsdoc` version: 36.1.0
- `@typescript-eslint/parser` version: 4.33.0

**Comment by brettz9** [comment]
@Ulrikop : Sorry it took longer than expected for me to get to it, but the other `@example` variety is now fixed too. (v37.1.0)
